### PR TITLE
feat(LandingPage): Creating components + initial styling

### PR DESCRIPTION
**Pull request by nsbtr**
- Created some components for common elements such as headings and body text
- Created "Half-Hero" component
- Started putting some modules on the landing page
 
Spacing/padding and stuff is not yet done, but just wanted to create this PR in case you have some feedback regarding how I've created the components so far (and to avoid creating a massive PR 😛)

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const Button = styled.button`
+  max-width: 180px;
+  font-family: ${props => props.theme.fonts.secondaryFont};
+  font-weight: ${props => props.theme.weights.secondaryBold};
+  font-size: 14px;
+  color: ${props => props.theme.colors.white};
+  background: ${props => props.theme.colors.orange};
+  text-transform: uppercase;
+  margin-top: 50px;
```
Comment (colindamelio) [inline src/components/Button.js]: I would maybe remove this since it could vary on usage

File: src/components/H2.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const H2 = styled.h1`
```
Comment (colindamelio) [inline src/components/H2.js]: perhaps we keep it consistent and by a `styled.h2`

File: src/components/H2.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
```
Comment (colindamelio) [inline src/components/H2.js]: feel free to removes these imports from the similar components

File: src/components/HeroWithContent.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  background: ${props => props.theme.colors.white};
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  ${mq.desktop`
+    flex-direction: row;
+  `};
+`;
+
+const Image = styled.div`
```
Comment (colindamelio) [inline src/components/HeroWithContent.js]: styled image?

Comment (johnathanfong) [reply src/components/HeroWithContent.js]: It's not meant to be an `img` tag, but perhaps we say `ContentWrapper` or something `HeroContent`

**Comment by nsbtr** [comment]
Thanks @johnathanfong & @colindamelio. Made the requested changes!

File: src/components/BodyText.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const BodyText = styled.p`
```
[the remark this answers is not in the document]
Comment (nsbtr) [reply src/components/BodyText.js]: Having this as a component allows us to use the theme variables. I don't think we're able to do this with injectGlobal.

File: src/routes/Home.js [hunk 6]
```diff
@@ -1,17 +1,30 @@
...
-        <h2>
-          Learn to code your own website in the heart of Bali. Apply for our
-          14-day retreat today!
-        </h2>
+        <LandingModule />
+        <Navigation />
+        <HeroWithContent src={devImage} width={50}>
+          <H2>Front-End Fundamentals</H2>
+          <BodyText>
+            With 10 days of in-class training, our goal is to ensure you have a
```
[the remark this answers is not in the document]
Comment (nsbtr) [reply src/routes/Home.js]: We'll definitely do that eventually! My main goal here was just to start getting the components created 🙂

File: src/components/BodyText.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (JesseNoseworthy) [reply src/components/BodyText.js]: We can, but they can just be imported as per a usual JSON import. WebPack should handle any duplicates when bundling.

File: src/routes/Home.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (JesseNoseworthy) [reply src/routes/Home.js]: :100: 

Just an approach that has saved me in the past!

**Pull request merged**